classes/Project/ProjectFactory.js

```diff
@@ -288,19 +292,22 @@ export default class ProjectFactory {
           id: `${process.env.SERVERURL}project/${copiedProject._id}/page/${database.reserveId()}`,
           label: page.label,
           target: page.target,
+          creator: copiedProject.creator,
           items: await Promise.all(pageData.items.map(async item => {
             return await fetch(item.id)
             .then(response => response.json())
             .then(async itemData => {
               const newItem = new Line({
                 id: `${process.env.SERVERURL}project/${copiedProject._id}/line/${database.reserveId()}`,
-                target: itemData.target,
-                body: itemData.body,
-                motivation: itemData.motivation,
+                type: itemData.type,
                 label: itemData.label,
-                type: itemData.type
+                motivation: itemData.motivation,
+                body: itemData.body,
+                target: itemData.target,
+                creator: copiedProject.creator
               })
-              return await newItem.update()
+              await newItem.update()
```

Comment:
If you change the `id` before updating, you will not be updating the old Line. We want to fork the Line in RERUM and then store the new `_id`. 

It may be useful to allow `update()` to take a parameter of the address to update, if you want to have the new ID already in hand. I'm not certain that we are built to do this all the way up at the moment.